params/config.go

```diff
@@ -342,6 +342,12 @@ func (c *ChainConfig) SetDefaults() {
 		logger.Warn("Override the default proposer update interval to the chain config", "interval",
 			c.Governance.Reward.ProposerUpdateInterval)
 	}
+
+	if c.Governance.Reward.MinimumStake.Cmp(common.Big0) == 0 {
+		c.Governance.Reward.MinimumStake = MinimumStakingAmount()
+		logger.Warn("Override the default minimum staking amount to the chain config", "amount",
+			c.Governance.Reward.MinimumStake)
+	}
```

Comment:
I think `0` is supposed to be allowed as a minimum stake.
Also, we need to check whether `storedcfg.Governance.Reward.MinimumStake` is nil.

Reply:
I have updated it. Thanks.